### PR TITLE
Feature/bot credentials

### PR DESCRIPTION
**Pull request by Opti213**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,12 +81,14 @@ $ pip install fastapi uvicorn
...
 * Create a file `main.py` with following content:
+
 ```python3
-from botx import Bot, ExpressServer, IncomingMessage, Message, Status
+from botx import Bot, BotXCredentials, IncomingMessage, Message, Status
 from fastapi import FastAPI
 from starlette.status import HTTP_202_ACCEPTED
 
-bot = Bot(known_hosts=[ExpressServer(host="cts.example.com", secret_key="secret")])
+bot = Bot(bot_accounts=[BotXCredentials(host="cts.example.com", secret_key="secret",
```
Comment (lxmnk) [inline README.md:90]:
```python
bot_accounts = [
    BotXCredentials(host="cts.example.com", secret_key="secret", bot_id=UUID("bot_id"))
]
bot = Bot(bot_accounts=bot_accounts)
```

File: botx/bots/bots.py [hunk 2]
```diff
@@ -119,13 +119,13 @@ async def execute_command(self, message: dict) -> None:
             message: incoming message to bot.
 
         Raises:
-            ServerUnknownError: raised if message was received from unregistered host.
+            UnknownBotError: raised if message was received from unregistered bot.
```
Comment (lxmnk) [inline botx/bots/bots.py:122]: `from unregistered bot` -> `from unregistered bot account`.

File: botx/bots/bots.py [hunk 3]
```diff
@@ -119,13 +119,13 @@ async def execute_command(self, message: dict) -> None:
...
         """
         logger.bind(botx_bot=True, payload=message).debug("process incoming message")
         msg = Message.from_dict(message, self)
-        for server in self.known_hosts:
-            if server.host == msg.host:
+        for account in self.bot_accounts:  # noqa: WPS503
```
Comment (lxmnk) [inline botx/bots/bots.py:126]:
```python
try:
    self.get_account_by_bot_id(msg.bot_id)
except UnknownBotError:
    raise
    
await self(msg)
```

File: botx/bots/mixins/requests/mixin.py [hunk 4]
```diff
@@ -63,20 +63,20 @@ async def call_method(  # noqa: WPS211
...
         """
         if not isinstance(method, Token):
-            host = cast(str, credentials.host if credentials else host)
             await _fill_token(
                 self.client,
-                host,
-                cast(UUID, credentials.bot_id if credentials else bot_id),
-                cast(CredentialsSearchProtocol, self).get_cts_by_host(host),
+                cast(CredentialsSearchProtocol, self).get_account_by_bot_id(
+                    credentials.bot_id if credentials else bot_id,  # type: ignore
```
Comment (lxmnk) [inline botx/bots/mixins/requests/mixin.py:69]: What error was hidden by comment?

Comment (Opti213) [reply botx/bots/mixins/requests/mixin.py:69]: excepted UUID, got Optional[UUID]

**Review by Nimond — APPROVED** [review]

**Review by lxmnk — APPROVED** [review]

**Pull request merged**